translations/player_turns.tex/cs.po

```diff
@@ -114,13 +114,14 @@ msgid ""
 "\n"
 msgstr ""
 "\\vspace*{\\lang_header_adjustment}\n"
+"\\iftoggle{printable}{\\vspace*{\\lang_header_adjustment}}{}\n"
```

Comment:
Why? Don't you want it correctly spaced at all times?

Reply:
This is just a workaround for printable mode where I had to decrease the space between sections a bit more to make the text fit to a single page. This was more to keep the aesthetics of the digital version than anything else

Reply:
Exactly, shouldn't the aesthetics be kept for both versions? If the text is too long, you shorten it, not manipulate spacing.

Reply:
I haven't look very closely at the source of other PRs, mostly at the pictures attached, you didn't do similar hacks there?